app/src/main/java/com/example/stromtracker/ui/amortisationsrechnerPV/AmortisationsrechnerPVFragment.kt

```diff
@@ -33,7 +33,7 @@ class AmortisationsrechnerPVFragment : Fragment() {
 
         editLeistung = root.findViewById(R.id.amortPV_edit_leistung)
         editAK = root.findViewById(R.id.amortPV_edit_anschaffung)
-        editErtrag = root.findViewById(R.id.amortPV_edit_kWh_pro_kWp)
+        editErtrag = root.findViewById(R.id.amortPV_edit_ertrag)
         editVerguetung = root.findViewById(R.id.amortPV_edit_vergütung)
         editEigenverbrauch = root.findViewById(R.id.amortPV_edit_eigenverbrauch)
         editPreisKwh = root.findViewById(R.id.amortPV_edit_preis_kwh)
```

Comment:
XML naming and ids should be in snake case only.

Reply:
This will be fixed in the Pull Request from AmortisationsrechnerPV.